src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -324,16 +324,7 @@ internal MemberAccessor MemberAccessorStrategy
             {
                 if (_memberAccessorStrategy == null)
                 {
-#if NETSTANDARD2_1
-                    if (RuntimeFeature.IsDynamicCodeSupported)
-                    {
-                        _memberAccessorStrategy = new ReflectionEmitMemberAccessor();
-                    }
-                    else
-                    {
-                        _memberAccessorStrategy = new ReflectionMemberAccessor();
-                    }
-#elif NETFRAMEWORK
+#if NETFRAMEWORK || NETCOREAPP
```

Comment:
Currently System.Text.Json does not target NetStandard2.1. If we did add that configuration then we could use `RuntimeFeature.IsDynamicCodeSupported` to determine the strategy, but adding a 2.1 configuration is likely not going to help with 5.0 since Xamarin and other will just target netcoreapp5.0 instead.

Reply:
Let's discuss outside this PR. Maybe we can still use `IsDynamicSupported` for the netcoreapp configs.